lib/diagnostic.md

```diff
@@ -54,7 +54,7 @@ Please write a method that traverses from the lister DOM node (one of the list
 item elements) to the body DOM node, then hides it.
 
 <!-- your answer starts here -->
-
+&('listerItem').grandparent $( "form" ).hide();
 <!-- your answer ends here -->
```

Comment:
$('.listerItem').parent().parent().hide()

Grandparent isn't a method.  If you are confused please google next time or refer to the docs